pages.pt-BR/common/ab.md

```diff
@@ -1,13 +1,12 @@
 # ab
-
-> Ferramenta de benchmarking do servidor Apache. A ferramenta mais simples para realizar testes de carga.
+> Ferramenta do servidor Apache para realizar benchmarking e testes de carga em servidores web.
```

Comment:
You removed the blank line after the title, please restore it :)

Reply:
Also, I thought we had [agreed](https://github.com/tldr-pages/tldr/pull/2941#discussion_r278220302) that the tool isn't specific to the Apache _server_. The english description we agreed on was

> Apache Benchmarking tool ...

So in Portuguese it would be something like

> Ferramenta da Apache ...

Am I missing something?

Reply:
:+1: